apps/service-portal/src/utils/userManager.ts

```diff
@@ -11,17 +11,18 @@ const settings: UserManagerSettings = {
   // eslint-disable-next-line @typescript-eslint/camelcase
   client_id: 'island-is-1',
   // eslint-disable-next-line @typescript-eslint/camelcase
-  // TODO: Pending support for Silent signin from Identity server
-  //silent_redirect_uri: `${window.location.origin}/silent/signin-oidc`,
+  silent_redirect_uri: `${window.location.origin}/silent/signin-oidc`,
   // eslint-disable-next-line @typescript-eslint/camelcase
   redirect_uri: `${window.location.origin}/signin-oidc`,
   // eslint-disable-next-line @typescript-eslint/camelcase
+  post_logout_redirect_uri: `${window.location.origin}`,
+  // eslint-disable-next-line @typescript-eslint/camelcase
   response_type: 'code',
   revokeAccessTokenOnSignout: true,
   loadUserInfo: true,
-  automaticSilentRenew: false,
-  scope: 'openid profile offline_access api_resource.scope',
-  userStore: new WebStorageStateStore({ store: new InMemoryWebStorage() }),
+  automaticSilentRenew: true,
+  scope: 'openid profile api_resource.scope',
+  userStore: new WebStorageStateStore({ store: window.sessionStorage }),
```

Comment:
Why the change to session storage? Wouldnt this be vulnarable to XSS?

Reply:
Potentially yes, and we should mitigate that risk. The upside is that we don't need to contact auth server on refresh.
Note that the in-memory storage also makes the tokens accessible in the browser.

Reply:
Yes but the risk I would guess is less when dealing with In-memory storage.  But maybe the upside outweighs the potentential risk. Am assuming this will or has been cleared with Syndis.

Reply:
Haven't cleared with Syndis but we should. As I see it the tokens are either stored in the browser or not, in-memory storage might be a bit more obscure but the tokens are still there.